common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -392,44 +449,47 @@ innovation_sentinel = {
 	flag = global_maa
 }
 
-innovation_snow_hunter = {
+innovation_silithid_colossus = {
 	group = culture_group_regional
```

Comment:
I would make kunchongs siege weapons. They're used to break down the Serpent's Wall.
https://wowpedia.fandom.com/wiki/Ik%27thik_Colossus
https://wowpedia.fandom.com/wiki/Raigonn

Reply:
We have no other idea for giants for Mantid

Reply:
> We have no other idea for giants for Mantid

I don't think it's a problem.